scripts/ci/local_ci_stop_environment.sh

```diff
@@ -45,7 +45,6 @@ export HOST_GROUP_ID
 
 docker-compose \
     -f "${MY_DIR}/docker-compose.yml" \
-    -f "${MY_DIR}/docker-compose-kubernetes.yml" \
```

Comment:
You can also remove the "docker-compose-kubernetes.yml" file

Reply:
Deleted.

Reply:
Good!